### PR TITLE
Week9 landing page

### PR DESCRIPTION
**Pull request by lwasser**

### CODE REVIEW

File: _posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md [hunk 1]
```diff
@@ -40,39 +39,149 @@ early!
...
-## Homework Plot 1 - Grid of 3 - 3 band CIR plots post fire
 
 
-{:.output}
-{:.display_data}
 
-<figure>
+{:.input}
+```python
+def stack_modis_bands(h4_path, crop_bound):
```
Comment (lwasser) [inline _posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md]: @nkorinek - you need to hide the code for these lessons. 
look at some of the other landing pages. you add 'hide' as a tag to the cell where you want to hide the code and just show the output. as submitted the answers to the homework as visible here

Comment (nkorinek) [reply _posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md]: @lwasser apologies! Just pushed up the corrected version to earth-analytics-lessons

File: _posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md [hunk 2]
```diff
@@ -30,34 +30,172 @@ Landsat data to calculate burn indices.
...
-{:.output}
-{:.display_data}
+Create a new `Jupyter Notebook` document. Name it: **lastName-firstInitial-weeknumber.ipynb**
+Within your `.ipynb` document, include the plots listed below. Submit both the
+`Jupyter Notebook`. Be sure to name your files as instructed above.
 
-<figure>
 
-<img src = "{{ site.url }}/images/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis/2017-01-01-week-09-spectral-remote-sensing-modis_12_0.png" alt = "CIR Composite images from NAIP, Landsat, and MODIS for the post-Cold Springs fire.">
-<figcaption>CIR Composite images from NAIP, Landsat, and MODIS for the post-Cold Springs fire.</figcaption>
```
Comment (lwasser) [inline _posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md]: @nkorinek  this is a figure caption. see how it says figcaption? YOu have to add a metadata element that has the caption for this to appear. we need a caption for each plot for accessibility!

File: _posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md [hunk 3]
```diff
@@ -66,20 +204,32 @@ early!
...
+## Homework Plot 1 - Grid of 3 - 3 band CIR plots post fire
+
+
 {:.output}
 {:.display_data}
 
 <figure>
 
-<img src = "{{ site.url }}/images/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis/2017-01-01-week-09-spectral-remote-sensing-modis_15_0.png" alt = "NBR images calculated from Landsat for pre- and post-Cold Springs fire.">
-<figcaption>NBR images calculated from Landsat for pre- and post-Cold Springs fire.</figcaption>
```
Comment (lwasser) [inline _posts/courses/earth-analytics-python/09-multispectral-remote-sensing-fire/2017-01-01-week-09-spectral-remote-sensing-modis.md]: here is another caption @nkorinek

**Comment by nkorinek** [comment]
@lwasser Captions added, code hid!

**Pull request merged**